content/bn/auto_scaling.md

```diff
@@ -0,0 +1,11 @@
+---
+title: অটোস্কেলিং (Auto Scaling)
+status: Completed
+category: সম্পত্তি
+---
+
+অটোস্কেলিং হল সাধারণত একটি সিস্টেমের কম্পিউটিং রিসোর্সগুলির পরিপ্রেক্ষিতে স্বয়ংক্রিয়ভাবে  [স্কেল](/স্কেলযোগ্যতা/)  ক্ষমতা। অটোস্কেলিং সিস্টেমের সাহায্যে, ব্যবহারকারী চাহিদা অনুযায়ী রিসোর্স স্কেল করতে এবং প্রয়োজনের সময়  স্বয়ংক্রিয়ভাবে রিসোর্সগুলি যোগ করতে পারে। অটোস্কেলিং প্রক্রিয়া পরিবর্তনশীল এবং মেমোরি বা প্রক্রিয়া সময়ের মতো বিভিন্ন মেট্রিক্সের উপর ভিত্তি করে স্কেল করার জন্য কনফিগারযোগ্য। ক্লাউড পরিষেবাগুলি সাধারণত অটোস্কেলিং এর মাধ্যমে পরিচালিত হয় কারণ বেশিরভাগ অন-প্রিমাইজ ডিপ্লোয়মেন্টের চেয়ে এটিতে আরও বেশি বাস্তবায়ন করার ব্যবস্থা রয়েছে।
+
+পূর্বে, সিস্টেমের  অবকাঠামো এবং অ্যাপ্লিকেশনগুলিকে আর্কিটেক্ট করা হয়েছিল সিস্টেমের সর্বোচ্চ ব্যবহার বিবেচনা করার মাধ্যমে। এই স্থাপনার দ্বারা রিসোর্সগুলি কম ব্যবহার হচ্ছিল এবং ভোক্তাদের চাহিদা পরিবর্তন করার জন্য স্থিতিস্থাপক ছিল। এই স্থিতিস্থাপকতা ফলে ব্যবসায় উচ্চ খরচ ছিল এবং অতিরিক্ত চাহিদা দেখা দিলেই ব্যবসা বন্ধ হয়ে যাচ্ছিল।
+
+ক্লাউড, [virtualizing](/virtualization/), and [containerizing](/containerization/) অ্যাপ্লিকেশন এবং তাদের নির্ভরতা ব্যবহার করে, সংস্থাগুলি ব্যবহারকারীর চাহিদা অনুযায়ী অ্যাপ্লিকেশন তৈরি করতে পারে। তারা অ্যাপ্লিকেশন চাহিদা নিরীক্ষণ করতে পারে এবং স্বয়ংক্রিয়ভাবে তাদের স্কেল করতে পারে, একটি সর্বোত্তম ব্যবহারকারীর অভিজ্ঞতা প্রদান করে। যেমন প্রতি শুক্রবার সন্ধ্যায় Netflix-এর দর্শক সংখ্যা বৃদ্ধি। স্বয়ংক্রিয়ভাবে আউট করার অর্থ হল গতিশীলভাবে আরও সংস্থান যোগ করা: উদাহরণস্বরূপ, আরও ভিডিও স্ট্রিমিংয়ের অনুমতি দেয় এমন সার্ভারের সংখ্যা বৃদ্ধি করা এবং একবার ব্যবহার স্বাভাবিক হয়ে গেলে আবার স্কেল করা।
```

Comment:
Can you localize these words `virtualizing`, `containerizing` ?

Reply:
I think this sentence can be modified `তারা অ্যাপ্লিকেশন চাহিদা নিরীক্ষণ করতে পারে এবং স্বয়ংক্রিয়ভাবে তাদের স্কেল করতে পারে, একটি সর্বোত্তম ব্যবহারকারীর অভিজ্ঞতা প্রদান করে`